config/v1/json.go

```diff
@@ -0,0 +1,42 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"regexp"
+)
+
+// reUUID4 uses a regexp to avoid taking a dependency on a UUID library for now
```

Comment:
apimachinery already has one: https://github.com/kubernetes/apimachinery/blob/master/pkg/util/uuid/uuid.go

Reply:
We don't actually depend on that, so that brings in a whole mess of new dependencies